DependencyInjection/MonologExtension.php

```diff
@@ -652,6 +652,13 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 $handler['app_name'],
             ));
             break;
+        case 'server_log':
+            $definition->setArguments(array(
+                $handler['host'],
```

Comment:
> And there should better be a nice exception being thrown if the new handler is not present.

It can occur if one use the very lastest version of the bundle, but with an older version symfony?